.ci/gcb-test-failure-ticket.yaml

```
@@ -14,6 +14,13 @@ steps:
       waitFor: ["collect-nightly-test-status"]
       args:
         - 'create-test-failure-ticket'
+    - name: 'gcr.io/graphite-docker-images/go-plus'
```

I think we should wait a bit to ensure that the github labeler had time to run

```suggestion
    - name: 'ubuntu'
      args: ['sleep', '120']
    - name: 'gcr.io/graphite-docker-images/go-plus'
```